malloc_conf_tests.py

```diff
@@ -0,0 +1,53 @@
+import subprocess
```

Comment:
re: name of the test, maybe include the keyword `stress_test` and avoid malloc_conf which is more of a detail here?

Reply:
Renamed it to `conf_stress_tests.py` mainly to separate it from the `stress_test` folder that already exists and highlight that various confs are the keys here. I am really bad at naming, please suggest if any better ideas.